flexmeasures/data/models/time_series.py

```diff
@@ -94,6 +110,7 @@ def search_beliefs(
             beliefs_before=beliefs_before,
             source=source,
             most_recent_only=most_recent_only,
+            most_recent_events_only=most_recent_events_only,
```

Comment:
It seems to me that `horizons_at_least` and `horizons_at_most` are not being passed through to `TimedBelief.search` here.

Reply:
Good catch!